<html><body><pre>
yarn run v1.15.2
$ eslint .

/home/jenkins/workspace/org_ui-plugin-find-contact_PR-45/project/test/bigtest/network/configs/categories.js
  1:32  error  Unable to resolve path to module '@folio/organizations/src/common/constants'  import/no-unresolved

/home/jenkins/workspace/org_ui-plugin-find-contact_PR-45/project/test/bigtest/network/configs/contacts.js
  1:30  error  Unable to resolve path to module '@folio/organizations/src/common/constants'  import/no-unresolved

✖ 2 problems (2 errors, 0 warnings)

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
</pre><body></html>

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,8 +2,16 @@
 
 ## (IN PROGRESS)
 
+## [2.0.0](https://github.com/folio-org/ui-plugin-find-contact/tree/v2.0.0) (2020-03-13)
+[Full Changelog](https://github.com/folio-org/ui-plugin-find-contact/compare/v1.4.0...v2.0.0)
+
+* bump the @folio/stripes peer to v3.0.0
+
 ### Stories
 * [UIORGS-112](https://issues.folio.org/browse/UIORGS-112) Sorting when adding contacts does not work
+
+### Bug Fixes
+* [UIPFCONT-3](https://issues.folio.org/browse/UIPFCONT-3) Security update eslint to >= 6.2.1 or eslint-util >= 1.4.1
 
 ## [1.4.0](https://github.com/folio-org/ui-plugin-find-contact/tree/v1.4.0) (2019-12-04)
 [Full Changelog](https://github.com/folio-org/ui-plugin-find-contact/compare/v1.3.0...v1.4.0)

--- a/findContact/FindContactContainer.js
+++ b/findContact/FindContactContainer.js
@@ -8,10 +8,10 @@ import {
   StripesConnectedSource,
 } from '@folio/stripes/smart-components';
 import { stripesConnect } from '@folio/stripes/core';
-import { transformCategoryIdsToLables } from '@folio/organizations/src/common/utils/category';
-import { categoriesResource } from '@folio/organizations/src/common/resources';
-import { CONTACTS_API } from '@folio/organizations/src/common/constants';
 
+import { transformCategoryIdsToLables } from './utils';
+import { categoriesResource } from './resources';
+import { CONTACTS_API } from './api';
 import filterConfig from './filterConfig';
 
 const INITIAL_RESULT_COUNT = 30;

--- a/findContact/api.js
+++ b/findContact/api.js
@@ -1,0 +1,2 @@
+export const CATEGORIES_API = 'organizations-storage/categories';
+export const CONTACTS_API = 'organizations-storage/contacts';

--- a/findContact/resources.js
+++ b/findContact/resources.js
@@ -1,0 +1,12 @@
+import {
+  baseManifest,
+} from '@folio/stripes-acq-components';
+
+import { CATEGORIES_API } from './api';
+
+// eslint-disable-next-line import/prefer-default-export
+export const categoriesResource = {
+  ...baseManifest,
+  records: 'categories',
+  path: CATEGORIES_API,
+};

--- a/findContact/utils.js
+++ b/findContact/utils.js
@@ -1,0 +1,10 @@
+// eslint-disable-next-line import/prefer-default-export
+export const transformCategoryIdsToLables = (categories, categoryIds = []) => {
+  const categoriesMap = (categories || []).reduce((acc, category) => {
+    acc[category.id] = category.value;
+
+    return acc;
+  }, {});
+
+  return categoryIds.map(categoryId => categoriesMap[categoryId] || '').join(', ');
+};

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@folio/plugin-find-contact",
-  "version": "1.4.0",
+  "version": "2.0.0",
   "description": "Find and select contacts plugin for Stripes",
   "repository": "folio-org/ui-plugin-find-contact",
   "publishConfig": {
@@ -14,8 +14,8 @@
     "actsAs": ["plugin"],
     "pluginType": "find-contact",
     "okapiInterfaces": {
-      "organizations-storage.categories": "1.0 1.2",
-      "organizations-storage.contacts": "2.0 2.1"
+      "organizations-storage.categories": "1.1",
+      "organizations-storage.contacts": "2.0"
     },
     "displayName": "ui-plugin-find-contact.meta.pluginTitle"
   },
@@ -46,8 +46,7 @@
     "regenerator-runtime": "^0.13.3"
   },
   "dependencies": {
-    "@folio/organizations": "^1.4.0",
-    "@folio/stripes-acq-components": "^1.1.0",
+    "@folio/stripes-acq-components": "^2.0.0",
     "lodash": "^4.17.11",
     "prop-types": "^15.6.0"
   },

--- a/test/bigtest/network/configs/categories.js
+++ b/test/bigtest/network/configs/categories.js
@@ -1,4 +1,4 @@
-import { CATEGORIES_API } from '@folio/organizations/src/common/constants';
+import { CATEGORIES_API } from '../../../../findContact/api';
 
 const configCategories = server => {
   server.get(CATEGORIES_API, (schema) => {

--- a/test/bigtest/network/configs/contacts.js
+++ b/test/bigtest/network/configs/contacts.js
@@ -1,4 +1,4 @@
-import { CONTACTS_API } from '@folio/organizations/src/common/constants';
+import { CONTACTS_API } from '../../../../findContact/api';
 
 const configContacts = server => {
   server.get(`${CONTACTS_API}`, (schema) => {

